src/Monolog/Handler/FluentHandler.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/*
+ * This file is part of the Monolog package.
+ *
+ * (c) Jordi Boggiano <j.boggiano@seld.be>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Monolog\Handler;
+
+use Fluent\Logger\FluentLogger;
+use Monolog\Logger;
+
+/**
+ * Handler to send messages to a Fluentd (http://www.fluentd.org/) server
+ *
+ * Requires https://github.com/fluent/fluent-logger-php
+ *
+ * @author Dai Akatsuka <d.akatsuka@gmail.com>
+ * @author Yegor Tokmakov <yegor@tokmakov.biz>
+ */
+class FluentHandler extends AbstractProcessingHandler
+{
+    /**
+     * @var FluentLogger
+     */
+    protected $logger;
+
+    /**
+     * Initialize Handler
+     *
+     * @param FluentLogger $logger
+     * @param bool|string $host
+     * @param int $port
+     * @param int $level
+     * @param bool $bubble
+     */
+    public function __construct(
+        $logger = null,
+        $host   = FluentLogger::DEFAULT_ADDRESS,
+        $port   = FluentLogger::DEFAULT_LISTEN_PORT,
+        $level  = Logger::DEBUG,
+        $bubble = true
+    )
+    {
+        parent::__construct($level, $bubble);
+
+        if (is_null($logger)) {
+            $logger = new FluentLogger($host, $port);
+        }
```

Comment:
Not sure if that really makes sense. It over-complicates the constructor and requires the user to pass values that will never be used just because they want to, for example, change the logging level.